# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../linux-header/hiddev.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

Fix inclusion by user-space packages, such as hal.

  - Rene Rebe <rene@exactcode.de>

--- usr/include/linux/hiddev.h.orig	2005-05-10 08:33:13.000000000 +0200
+++ usr/include/linux/hiddev.h	2005-05-10 08:27:49.000000000 +0200
@@ -33,6 +33,10 @@
  * The event structure itself
  */
 
+#ifndef HID_MAX_USAGES
+#define HID_MAX_USAGES 8
+#endif
+
 struct hiddev_event {
 	unsigned hid;
 	signed int value;
